src/formatDate.js

```diff
@@ -1,14 +1,38 @@
 'use strict';
 
-/**
- * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
- *
- * @returns {string}
- */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat.splice(3, 1);
```

Comment:
The `splice` method is used incorrectly here. `splice` is an array method, but `fromFormat` is expected to be an array of strings. You should use `split` to separate the format string into components if needed.